sympy/ntheory/factor_.py

```diff
@@ -311,6 +315,59 @@ def multiplicity(p, n):
     return m
 
 
+def multiplicity_in_factorial(b, n):
+    """
+    Finds the greatest integer m such that b**m divides factorial of n without calculating the factorial
+
+    Examples
+    ========
+
+    >>> from sympy.ntheory import multiplicity_in_factorial
+
+    >>> multiplicity_in_factorial(5, 60)
```

Comment:
Think about what information is being conveyed with each example. What do you want to demonstrate? Some text and fewer examples might be better. I think the following is too much. But an example showing that this function can be called directly or that `multiplicity` can be called with an unevaluated `factorial` would be good.
_________

It is not necessary to compute n! when wanting to determine the multiplicity of a number p in n!. For example, the multiplicity of 10 in n! will tell the number of trailing zeros in n! and a trailing 0 will be introduced every time a new factor of 5 is encountered. For example, there are 6 factors of 5 in the 25! (1 in 5, 10, 15, 20 and 2 in 25) so there are 6 trailing zeros in 25!
```python
>>> factorial(25)
15511210043330985984000000
```
In general, the multiplicity of p in n! is ``(n - sum of digits of n in base p)/(p - 1)``. So if we wanted to know how many zeros are at the end of 4200! factorial we can just compute that sum (instead of 4200! -- which has over 13000 digits):

```python
>>> digits(4200,5)
[5, 1, 1, 3, 3, 0, 0]
```
The first element is the base so the sum is 8: there are (4200 - 8)/(5 - 1) = 1048 factors of 5 (and many more factors of 2) so 4200! ends with 1048 zeros

```python
>>> multiplicity_in_factorial(10, 4200)
1048
```
If you pass an unevaluated factorial to ``multiplicity`` you will get the same answer. You will get the same answer if you evaluate it, but it will take much longer:
```python
>>> multiplicity(10, factorial(4200, evaluate=False))  # TODO - make multiplicity recognize this case
1048
```